monkey/common/common_consts/validation_formats.py

```diff
@@ -1,3 +1,5 @@
 # Defined in UI on ValidationFormats.js
 IP_RANGE = "ip-range"
 IP = "ip"
+VALID_DIR_LINUX = "valid-directory-linux"
+VALID_DIR_WINDOWS = "valid-directory-windows"
```

Comment:
Same as the other comment, the name is misleading as this validator doesn't check if the directory is valid, it only checks if the start of the path looks like the start of an absolute path.